flycheck.el

```diff
@@ -521,20 +521,43 @@ enabled.  Changing it will not affect buffers which already have
   :package-version '(flycheck . "0.15")
   :safe #'booleanp)
 
+(defconst flycheck-minimum-level-choices
```

Comment:
@cpitclaudel I thought that you could easily define widgets that construct their `:type` dynamically, but going back to the docs I see that this is rather convoluted—it requires a custom `:create` function I think.  And I didn't think of the auto-loading issue.

You're right, let's leave it as is.  It's a good compromise between convenience and extensibility, and above all, it's simple :)